data/tlg5023/__cts__.xml

```diff
@@ -0,0 +1,3 @@
+<ti:textgroup xmlns:ti="http://chs.harvard.edu/xmlns/cts" urn="urn:cts:greekLit:tlg5023">
+  <ti:groupname xml:lang="lat">Scholia Graeca</ti:groupname>
```

Comment:
The TLG has "Scholia in Euripidem" as the author. I think it makes sense to use that.